doc/admin-api.md

```diff
@@ -714,16 +747,4 @@ Date: Thu, 26 Dec 2019 04:19:34 GMT
 Content-Type: text/plain
 ```
 
-[Back to TOC](#Table-of-Contents)
-
-*API*：/apisix/admin/plugins/?all=true
```

Comment:
The documentation show the behavior of the master branch code, not the release version. We are planning to maintain different documentation for different release in their branches. Anyway, the master branch of documentation should be corresponding with the master branch of code.

Reply:
I found it. I'll revise it